api/src/main/resources/notebooks.yaml

```diff
@@ -1,10 +1,7 @@
 # This file is a hand-crafted combination of Leo and its proxy APIs.
 #  - A subset of Welder (modified for cluster/path prefix): https://github.com/DataBiosphere/welder/blob/master/server/src/main/resources/api-docs.yaml
```

Comment:
"cluster/path" prefix.  Also update the many cluster references in the "subset" Jupyter and Welder sections themselves